litellm/model_prices_and_context_window_backup.json

```diff
@@ -89,6 +89,156 @@
+        "input_cost_per_token_batches": 0.000001,
+        "output_cost_per_token_batches": 0.000004,
+        "litellm_provider": "openai",
+        "mode": "chat",
+        "supports_function_calling": true,
+        "supports_parallel_function_calling": true,
+        "supports_response_schema": true,
+        "supports_vision": true,
+        "supports_prompt_caching": true,
+        "supports_system_messages": true,
+        "supports_tool_choice": true,
+        "supports_web_search": true,
+        "search_context_cost_per_query": {
+            "search_context_size_low": 0.030,
+            "search_context_size_medium": 0.035,
+            "search_context_size_high": 0.050
+        }
+    },
+    "gpt-4.1-mini": {
+        "max_tokens": 1047576,
+        "max_input_tokens": 1047576,
+        "max_output_tokens": 32768,
+        "input_cost_per_token": 0.0000004,
+        "output_cost_per_token": 0.0000016,
+        "cache_read_input_token_cost": 0.0000001,
+        "input_cost_per_token_batches": 0.0000002,
+        "output_cost_per_token_batches": 0.0000008,
+        "litellm_provider": "openai",
+        "mode": "chat",
+        "supports_function_calling": true,
+        "supports_parallel_function_calling": true,
+        "supports_response_schema": true,
+        "supports_vision": true,
+        "supports_prompt_caching": true,
+        "supports_system_messages": true,
+        "supports_tool_choice": true,
+        "supports_web_search": true,
+        "search_context_cost_per_query": {
+            "search_context_size_low": 0.025,
+            "search_context_size_medium": 0.0275,
+            "search_context_size_high": 0.030
+        }
+    },
+    "gpt-4.1-mini-2025-04-14":{
+        "max_tokens": 1047576,
+        "max_input_tokens": 1047576,
+        "max_output_tokens": 32768,
+        "input_cost_per_token": 0.0000004,
+        "output_cost_per_token": 0.0000016,
+        "cache_read_input_token_cost": 0.0000001,
+        "input_cost_per_token_batches": 0.0000002,
+        "output_cost_per_token_batches": 0.0000008,
+        "litellm_provider": "openai",
+        "mode": "chat",
+        "supports_function_calling": true,
+        "supports_parallel_function_calling": true,
+        "supports_response_schema": true,
+        "supports_vision": true,
+        "supports_prompt_caching": true,
+        "supports_system_messages": true,
+        "supports_tool_choice": true,
+        "supports_web_search": true,
+        "search_context_cost_per_query": {
+            "search_context_size_low": 0.025,
+            "search_context_size_medium": 0.0275,
+            "search_context_size_high": 0.030
+        }
+    },
+    "gpt-4.1-nano": {
+        "max_tokens": 1047576,
+        "max_input_tokens": 1047576,
+        "max_output_tokens": 32768,
+        "input_cost_per_token": 0.0000001,
+        "output_cost_per_token": 0.0000004,
+        "cache_read_input_token_cost": 0.000000025,
```

Comment:
Should be 0.03/100000=0.0000003?

Reply:
fixed, thanks

Reply:
sorry i think it should be 0.00000003 (7 zeros)